sys/random/prng_tinymt32.c

```diff
@@ -35,6 +36,21 @@ uint32_t random_uint32(void)
     return tinymt32_generate_uint32(&_random);
 }
 
+size_t random_bytes(uint8_t *buf, size_t size)
+{
+  size_t count = 0;
+  uint32_t last;
+
+  while(size > sizeof(uint32_t)) {
+    *((uint32_t*)(buf+count)) = tinymt32_generate_uint32(&_random);
```

Comment:
Thanks for the hint. Correct me if i'm wrong, this could be resolved with a  simple `__packed`, right? If there's a better practice, i would be happy to hear about it.